cloud-provider-snapshots/README.md

```diff
@@ -11,3 +11,28 @@ Next steps:
 3. Implement your resource handlers by adding code to provision your resources in your resource handler's methods.
 
 Please don't modify files `model.go` and `main.go`, as they will be automatically overwritten.
+
```

Comment:
Lines 3-13 look auto-generated from the sdk.  Please remove but keep line 1, the title.